packages/theme-chalk/src/carousel.scss

```diff
@@ -97,6 +97,14 @@
       }
     }
 
+    @include m(left) {
+      left: 0;
+    }
```

Comment:
`indicators` seem to have no left related class name.

Reply:
Deleted the relevant prop code before, this part forgot to delete